cloud/shared/bin/doctor_impl

```diff
@@ -79,7 +76,7 @@ class AwsCliCheck(Check):
         return "AWS CLI"
 
     def should_skip(self):
-        return self._get_cloud_provider() in ['aws', 'azure']
+        return self._get_cloud_provider() != 'aws'
```

Comment:
issue (blocking): the AWS CLI is needed for Azure deployments as well since our only azure deployment atm uses AWS SES

Reply:
Ah good to know! Reverted, and added a comment to clarify.

Reply:
Actually the original check was missing a "not". Added that.